src/components/ItaliaTheme/Pagination/Pagination.jsx

```diff
@@ -104,7 +104,7 @@ class Pagination extends Component {
                   onClick={this.handleItemClick}
                   type={type}
                   ellipsisItem={ellipsisItem}
-                  isServiceLink={isServiceLink}
+                  isServiceLink={type === 'prevItem' ? '' : isServiceLink}
```

Comment:
`isServiceLink` dovrebbe essere un valore booleano
```suggestion
                  isServiceLink={type !== 'prevItem' && isServiceLink}
```